### PR TITLE
Accept runners by fully qualified name.

### PR DESCRIPTION
**Pull request by robertwb**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by robertwb** [comment]
R: @charlesccychen

File: sdks/python/apache_beam/runners/runner.py [hunk 1]
```diff
@@ -41,26 +41,24 @@ def create_runner(runner_name):
...
+    module, runner = runner_name.rsplit('.', 1)
+    return getattr(__import__(module, {}, {}, [runner], -1), runner)()
   else:
-    raise RuntimeError(
+    raise ValueError(
         'Unexpected pipeline runner: %s. Valid values are '
-        'DirectPipelineRunner, DataflowPipelineRunner, EagerPipelineRunner, or '
-        'BlockingDataflowPipelineRunner.' % runner_name)
+        'DirectPipelineRunner, DataflowPipelineRunner, EagerPipelineRunner, '
+        'BlockingDataflowPipelineRunner or the fully qualified name of '
```
Comment (charlesccychen) [inline sdks/python/apache_beam/runners/runner.py]: You dropped "DiskCachedPipelineRunner".  Can we refactor out the tuples above into variables and use them in constructing this list of available runners?

Comment (robertwb) [inline sdks/python/apache_beam/runners/runner.py]: Well, DiskCachedPipelineRunner was never in the list. Good idea to pull them out though. Done.

File: sdks/python/apache_beam/runners/runner.py [hunk 2]
```diff
@@ -40,25 +45,20 @@ def create_runner(runner_name):
...
     module, runner = runner_name.rsplit('.', 1)
     return getattr(__import__(module, {}, {}, [runner], -1), runner)()
   else:
     raise ValueError(
-        'Unexpected pipeline runner: %s. Valid values are '
-        'DirectPipelineRunner, DataflowPipelineRunner, EagerPipelineRunner, '
-        'BlockingDataflowPipelineRunner or the fully qualified name of '
-        'a PipelineRunner subclass.' % runner_name)
+        'Unexpected pipeline runner: %s. Valid values are %s'
+        'or the fully qualified name of a PipelineRunner subclass.' % (
```
Comment (charlesccychen) [inline sdks/python/apache_beam/runners/runner.py]: space before "or" or after "%s" on the previous line

**Comment by charlesccychen** [comment]
Thanks! LGTM.

File: sdks/python/apache_beam/runners/runner.py (hunk 2 above)
Comment (robertwb) [inline sdks/python/apache_beam/runners/runner.py]: Nice catch.

**Pull request closed without merging**